compile error: no method named `set_max_frame_size`

Projects that depend on h2 can no longer compile due to accessing an unstable feature from the stable API:

```
error[E0599]: no method named `set_max_frame_size` found for type `codec::framed_read::FramedRead<codec::framed_write::FramedWrite<T, _>>` in the current scope
  --> /Users/ver/b/rs/h2/src/codec/mod.rs:53:15
   |
53 |         inner.set_max_frame_size(max_frame_size);
   |               ^^^^^^^^^^^^^^^^^^

   Compiling ring v0.11.0
error[E0599]: no method named `set_max_recv_frame_size` found for type `codec::Codec<T, _>` in the current scope
   --> /Users/ver/b/rs/h2/src/client.rs:213:19
    |
213 |             codec.set_max_recv_frame_size(max as usize);
    |                   ^^^^^^^^^^^^^^^^^^^^^^^

error[E0599]: no method named `set_max_recv_frame_size` found for type `codec::Codec<T, _>` in the current scope
  --> /Users/ver/b/rs/h2/src/server.rs:97:19
   |
97 |             codec.set_max_recv_frame_size(max as usize);
   |                   ^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 3 previous errors

error: Could not compile `h2`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

The public Client and Server interfaces attempt to call `codec.set_max_recv_frame_size`:

```rust
impl<T, B: IntoBuf> Future for Handshake<T, B>
where
    T: AsyncRead + AsyncWrite,
{
    type Item = Client<T, B>;
    type Error = ::Error;

    fn poll(&mut self) -> Poll<Self::Item, Self::Error> {
        let (io, _) = try_ready!(self.inner.poll());

        debug!("client connection bound");

        // Create the codec
        let mut codec = Codec::new(io);

        if let Some(max) = self.settings.max_frame_size() {
            codec.set_max_recv_frame_size(max as usize);
        }
```

```rust
impl<T, B> Server<T, B>
where
    T: AsyncRead + AsyncWrite + 'static,
    B: IntoBuf + 'static,
{
    fn handshake2(io: T, settings: Settings) -> Handshake<T, B> {
        // Create the codec
        let mut codec = Codec::new(io);

        if let Some(max) = settings.max_frame_size() {
            codec.set_max_recv_frame_size(max as usize);
        }
```

The `Codec::new` (which must be stable, since it's called from client and server stable APIs above) calls the unstable `FramedRead::set_max_frame_size`.

```rust
#[derive(Debug)]
pub struct Codec<T, B> {
    inner: FramedRead<FramedWrite<T, B>>,
}

impl<T, B> Codec<T, B>
where
    T: AsyncRead + AsyncWrite,
    B: Buf,
{
    /// Returns a new `Codec` with the default max frame size
    #[inline]
    pub fn new(io: T) -> Self {
        Self::with_max_recv_frame_size(io, frame::DEFAULT_MAX_FRAME_SIZE as usize)
    }

    /// Returns a new `Codec` with the given maximum frame size
    pub fn with_max_recv_frame_size(io: T, max_frame_size: usize) -> Self {
        ...

        let mut inner = FramedRead::new(delimited);

        // Use FramedRead's method since it checks the value is within range.
        inner.set_max_frame_size(max_frame_size);

        Codec {
            inner,
        }
    }
}


impl<T, B> Codec<T, B> {
    /// Updates the max received frame size.
    ///
    /// The change takes effect the next time a frame is decoded. In other
    /// words, if a frame is currently in process of being decoded with a frame
    /// size greater than `val` but less than the max frame size in effect
    /// before calling this function, then the frame will be allowed.
    #[cfg(feature = "unstable")]
    #[inline]
    pub fn set_max_recv_frame_size(&mut self, val: usize) {
        self.inner.set_max_frame_size(val)
    }
```

```rust
impl<T> FramedRead<T> {
    ...

    /// Updates the max frame size setting.
    ///
    /// Must be within 16,384 and 16,777,215.
    #[cfg(feature = "unstable")]
    #[inline]
    pub fn set_max_frame_size(&mut self, val: usize) {
        assert!(
            frame::DEFAULT_MAX_FRAME_SIZE as usize <= val &&
                val <= frame::MAX_MAX_FRAME_SIZE as usize
        );
        self.inner.set_max_frame_length(val)
    }
}
```

--- a/src/codec/framed_read.rs
+++ b/src/codec/framed_read.rs
@@ -232,7 +232,6 @@ impl<T> FramedRead<T> {
     /// Updates the max frame size setting.
     ///
     /// Must be within 16,384 and 16,777,215.
-    #[cfg(feature = "unstable")]
     #[inline]
     pub fn set_max_frame_size(&mut self, val: usize) {
         assert!(DEFAULT_MAX_FRAME_SIZE as usize <= val && val <= MAX_MAX_FRAME_SIZE as usize);

--- a/src/codec/mod.rs
+++ b/src/codec/mod.rs
@@ -65,7 +65,6 @@ impl<T, B> Codec<T, B> {
     /// words, if a frame is currently in process of being decoded with a frame
     /// size greater than `val` but less than the max frame size in effect
     /// before calling this function, then the frame will be allowed.
-    #[cfg(feature = "unstable")]
     #[inline]
     pub fn set_max_recv_frame_size(&mut self, val: usize) {
         self.inner.set_max_frame_size(val)

